.travis.yml

```diff
@@ -165,24 +166,24 @@ install:
       if [[ ! $skip && $deps ]]; then mv composer.json.phpunit composer.json; fi
 
       if [[ ! $skip && $PHP = 7.* ]]; then
-          ([[ $deps ]] && cd src/Symfony/Component/HttpFoundation; composer require --dev --no-update mongodb/mongodb)
+          ([[ $deps ]] && cd src/Symfony/Component/HttpFoundation; composer require --dev --no-update mongodb/mongodb || true)
       fi
 
-    - if [[ ! $skip ]]; then $COMPOSER_UP; fi
-    - if [[ ! $skip ]]; then ./phpunit install; fi
+    - if [[ ! $skip ]]; then $COMPOSER_UP || true; fi
+    - if [[ ! $skip ]]; then ./phpunit install || true; fi
     - |
       # phpinfo
-      if [[ ! $PHP = hhvm* ]]; then php -i; else hhvm --php -r 'print_r($_SERVER);print_r(ini_get_all());'; fi
+      if [[ ! $PHP = hhvm* ]]; then php -i || true; else hhvm --php -r 'print_r($_SERVER);print_r(ini_get_all());'; fi
```

Comment:
does `php -i` really fail ? If yes, it means that the PHP shipped in Travis is totally broken, and there is no hope.

Reply:
it does not fail since it shows the `phpinfo();` but it seems that it triggers a segfault